addon/object/computed.js

```diff
@@ -1,10 +1,11 @@
 import Ember from 'ember';
-
 import {
   decoratedPropertyWithRequiredParams,
   decoratedPropertyWithOptionalCallback
 } from '../utils/decorator-macros';
 
+const { assert } = Ember;
```

Comment:
@pzuraq - There is no ember-cli-shims version requirement. As long as ember-decorators depends on `ember-cli-babel@^6.6.0` we get [the polyfill](https://github.com/ember-cli/babel-plugin-ember-modules-api-polyfill) for free.  This means that

```js
import { inject } from "@ember/service"
```

is transpiled back to 

```js
const inject = Ember.inject.service;
```

Which is basically what we are doing today (with the destructuring).  This change simply lets this addon prepare for consuming Ember in a way that will allow actual tree shaking down the line...